service/src/main/kotlin/fi/espoo/evaka/daycare/service/AbsenceService.kt

```diff
@@ -450,12 +470,12 @@ private fun Database.Read.getGroupReservations(groupId: GroupId, dateRange: Fini
 WITH all_placements AS (
   $placementsQuery
 )
-SELECT r.child_id, r.start_time, r.end_time FROM attendance_reservation r
-WHERE :dateRange && daterange((r.start_time at time zone 'Europe/Helsinki')::date, (r.end_time at time zone 'Europe/Helsinki')::date, '[]')
+SELECT r.child_id, r.date, r.start_time, r.end_time FROM attendance_reservation r
+WHERE :dateRange @> r.date
```

Comment:
This should probably use `between_start_and_end(:dateRange, r.date)` to have a chance of using an index